packages/gatsby/src/commands/build-javascript.ts

```diff
@@ -1,10 +1,18 @@
-/* @flow */
-const webpack = require(`webpack`)
-const webpackConfig = require(`../utils/webpack.config`)
+import webpack from "webpack"
+import webpackConfig from "../utils/webpack.config"
+import { IProgram } from "./types"
+import { Span } from "opentracing"
 
-const { reportWebpackWarnings } = require(`../utils/webpack-error-utils`)
+import { reportWebpackWarnings } from "../utils/webpack-error-utils"
 
-module.exports = async (program, { parentSpan }) => {
+export const buildProductionBundle = async (
+  program: IProgram,
+  {
+    parentSpan,
+  }: {
+    parentSpan: Span
+  }
```

Comment:
can we refactor this to not be wrapped in an object. I've seen this pattern in other places and we should move away from it as it's superfluous.